testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -198,7 +198,13 @@ class MiniAppDisplayActivity : BaseActivity() {
                     accessTokenScope: AccessTokenScope,
                     onSuccess: (tokenData: TokenData) -> Unit,
                     onError: (tokenError: MiniAppAccessTokenError) -> Unit
-            ) = onSuccess(AppSettings.instance.tokenData)
+            ) {
+                if(AppSettings.instance.accessTokenError != null){
+                    onError(AppSettings.instance.accessTokenError!!)
+                }else{
```

Comment:
I think, we can format in here, also in Line: 202 e.g. adding _SPACE_ at both side of `else`